http.js

```diff
@@ -37,6 +37,8 @@ module.exports = function probeHttp(options, _callback) {
           if (length && length.match(/^\d+$/)) {
             result.length = +length;
           }
+
+          result.responseUrl = res.request.uri.href;
```

Comment:
Is `res.request.uri` always defined in this place? It's common to get unexpected exceptions on deep properties access. I think, not i this case, but i should be sure.